packages/@uppy/compressor/src/index.js

```diff
@@ -53,14 +53,20 @@ export default class Compressor extends BasePlugin {
           totalCompressedSize += compressedSavingsSize
           const { name, type, size } = compressedBlob
           const extension = name && getFileNameAndExtension(name).extension
+          // Name might have been changed in file.meta, so we update only the extension
+          const newMetaName = `${getFileNameAndExtension(file.meta.name).name}.${extension}`
```

Comment:
We can't reuse the previously called `getFileNameAndExtension`?

Reply:
It uses `name` from the blob, while here I'm trying to use the `file.meta.name` from the metadata which might have been changed by the user. So different arguments to the function, can’t reuse, unless I'm seeing it wrong?